.github/workflows/ci.yml

```diff
@@ -52,3 +58,57 @@ jobs:
       - run: bundle exec rails db:schema:load
       - name: Run tests
         run: bundle exec rake
+  deploy_to_staging:
```

Comment:
I was going to suggest DRY'ing this up by creating a `install_deployment_dependencies` step which does the cloud foundry / Bundle steps whether we're deploying to Staging or Production. But then I remembered we can't re-use dependencies between steps.

I also wondered if there was a native way to alias different shared steps, but it looks like [GitHub is still working on that](https://github.com/actions/runner/issues/646). So this looks about as good as it's going to get!